webpack/redux/reducers/helpers.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/prefer-default-export
+export function parseMissingPermissions(errMsg) {
```

Comment:
can you use an arrow function? 
 ``` js
export const parseMissingPermissions = errMsg => { //some code}

Reply:
@amirfefer  I can, I guess I just used a regular function because I wasn't worried about the value of `this`.  I guess I didn't realize the other helper files have arrow functions.  They're prettier anyway so I will change it :)